src/config/backpack/operations/list.php

```diff
@@ -49,6 +49,16 @@
     // Nest action buttons within a dropdown in actions column
     'lineButtonsAsDropdown' => false,
 
+    // What is the minimum actions for the dropdown to be created
+    // Example: when minimum to drop is «2»,  any row with less than «2» action buttons
+    // will not create a dropdown, but will show the buttons inline
+    'lineButtonsAsDropdownMinimumToDrop' => 1,
```

Comment:
I think this is simpler and just as expressive:

```suggestion
    'lineButtonsAsDropdownMinimum' => 1,
```